src/mlstacks/analytics/client.py

```diff
@@ -34,6 +34,7 @@
 logger = getLogger(__name__)
 
 analytics.write_key = "tU9BJvF05TgC29xgiXuKF7CuYP0zhgnx"
+analytics.max_retries = 0
```

Comment:
Lowering this number makes a lot of sense. Considering the timeout is set to 15 seconds by default, in case of a timeout, it should never take that long. Perhaps 3 retries or 5 might be more suitable here.

Reply:
Hello @bcdurak, I appreciate the review and the insights. It's worth noting that the CLI hung for ~10 minutes before exiting with the default settings, likely due to exponential backoff.

I just tested it with `max_tries` set to lower values, and here's what I got:
`3`: 5 seconds
`5`: 20 seconds
`6`: 40 seconds

Setting it to 5 sounds like a balanced approach, as you suggested. I'll make the change.